include/mbedtls/platform_time.h

```diff
@@ -39,6 +39,28 @@ typedef MBEDTLS_PLATFORM_TIME_TYPE_MACRO mbedtls_time_t;
 typedef time_t mbedtls_time_t;
 #endif /* MBEDTLS_PLATFORM_TIME_TYPE_MACRO */
 
+#if defined(MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO)
+typedef MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO mbedtls_ms_time_t;
+#else
+#include <stdint.h>
+typedef int64_t mbedtls_ms_time_t;
+#endif /* MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO */
+
+/**
+ * \brief   Get time in milliseconds.
+ *
+ * \return Current time in milliseconds which is monotonically increasing.
+ *
+ * \note If MBEDTLS_PLATFORM_MS_TIME_ALT defined, users can provide their own
+ *       implementation.
+ *
+ * \warning This function is used for time difference only. The start time is
+ *          not defined. A well defined time function is not required in
+ *          TLS negotiation.
```

Comment:
"Time difference" suggests subtraction, which is not what this function does.

The usual way to specify this is:

```
This function returns a monotonically-increasing time value from a start time
that will differ from platform to platform, and possibly from run to run of
the process.
```

It's probably not relevant to mention the TLS requirement in the documentation here